.github/workflows/lint.yml

```diff
@@ -13,23 +13,18 @@ jobs:
     env:
       BUNDLE_WITHOUT: ruby_shadow:omnibus_package
     steps:
-    - uses: actions/checkout@v3
-    - uses: ruby/setup-ruby@v1
-      with:
-        ruby-version: 3.1
-        bundler-cache: false
-    - uses: r7kamura/rubocop-problem-matchers-action@v1  # this shows the failures in the PR
-    - run: |
-        gem install chefstyle
-        chefstyle -c .rubocop.yml
+      - uses: actions/checkout@v3
```

Comment:
@Jason3S can you restore the original indentation to prevent any downstream merge conflict weirdness with lines 16-24?

Reply:
done.